content/blaze.md

```diff
@@ -0,0 +1,559 @@
+
+```html
+{{> listsShow todosReady=Template.subscriptionsReady
+  list=(getFullList listIdOnly) todos=listIdOnly.todos}}
+```
+
+In this case, the `listShow` component can expect a data context of the form:
+
+```js
+{
+  todosReady: ...,
+  list: ...,
+  todos: ...
+}
+```
+
+<h3 id="helpers-in-tags">Helpers in tags</h3>
+We saw above that using a helper (or data context lookup) in the form `checked={{todo.checked}}` will add the checked property to the HTML tag if `todo.checked` evaluates to true. Also, you can directly include an object in the attribute list of an HTML element to set multiple attributes at once:
+
+```html
+<a {{attributes}}>My Link</a>
+```
+
+```js
+Template.foo.helpers({
+  attributes() {
+    return {
+      class: 'A class',
+      style: {background: 'blue'}
+    };
+  }
+});
+```
+
+<h3 id="rendering-html">Rendering pure HTML</h3>
+Although by default a mustache tag won't render any HTML, for [XSS](https://en.wikipedia.org/wiki/Cross-site_scripting) security reasons, you can render pure HTML with the triple-mustache: `{{{`.
+
+```html
+{{{myHtml}}}
+```
+
+```js
+Template.foo.helpers({
+  myHtml() {
+    return '<h1>This H1 will render</h1>';
+  }
+});
+```
+
+You should be extremely careful about doing this, and always ensure you aren't returning user-generated content (or escape it if you do!) from such a helper.
+
+<h3 id="block-helpers">Block Helpers</h3>
+A block helper, called with `{{#` is a helper that takes (and may render) a block of Spacebars. For instance, we saw the `{{#each in}}` helper above which repeats a given block of Spacebars once per item in a list. You can also render a *component* as a block helper, rendering it's content via the `Template.contentBlock` and `Template.elseBlock`. For instance, you could create your own `{{#if}}` helper with:
+
+```html
+<template name="myIf">
+  {{#if condition}}
+    {{> Template.contentBlock}}
+  {{else}}
+    {{> Template.elseBlock}}
+  {{/if}}
+</template>
+
+<template name="caller">
+  {{#myIf condition=true}}
+    <h1>I'll be rendered!</h1>
+  {{/myIf}}
+</template>
+```
+
+<h3 id="builtin-block-helpers">Builtin Block Helpers</h3>
+There are a few builtin block helpers that are worth knowing about:
+
+<h4 id="if-unless">If / Unless</h4>
+
+The `{{#if}}` and `{{#unless}}` helpers are fairly straightforward but invaluable for controlling the control flow of a template. Both operate by evaluating and checking their single argument for "falsey"-ness (in JS this means `null`, `undefined`, `0`, `''`, `[]` and of course `false`).
+
+```html
+{{#if something}}
+  <p>It's true</p>
+{{else}}
+  <p>It's false</p>
+{{/if}}
+```
+
+<h4 id="each-in">Each-in</h4>
+
+The `{{#each in}}` helper is a convenient way to step over a list whilst retaining the outer data context. 
```

Comment:
Should we add an example of a two-dimensional iteration? That's the main strength of `each in` over old `each`.